test/test_jit.py

```diff
@@ -1327,12 +1327,13 @@ def forward(self, x):
         FileCheck().check("quantize_linear").check_next("int_repr") \
                    .check_next("dequantize_linear") \
                    .check("conv2d").check_next("Constant") \
-                   .check_next("Constant").check_next("quantize_linear") \
-                   .check_next("int_repr").check_next("dequantize_linear") \
-                   .run(str(trace.graph))
+                   .check_next("Constant").check_next("Constant") \
```

Comment:
This test is too brittle because it expects to see Constant nodes at some specific spots. They could be moved, for instance, to the beginning of the graph and the IR would still be correct, but the test would fail. How could we fix this?

Reply:
hmm that might be hard because i cannot skip nodes in this check_* iteration. should i remove the const nodes completely from the test

Reply:
hmm i figured a way to remove const nodes, as this test checks the q-dq nodes pattern inserted